custom_model_runner/datarobot_drum/drum/adapters/cli/shared/drum_input_file_adapter.py

```diff
@@ -9,7 +9,7 @@
 from typing import Optional
 
 import pandas as pd
-from datarobot_drum.drum.enum import LOGGER_NAME_PREFIX
+from datarobot_drum.drum.enum import LOGGER_NAME_PREFIX, TEXT_GENERATION_PRED_COLUMN
```

Comment:
seems like this import is not used

Reply:
I will wait for other comments and will fix it with them